infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutConceptDao.kt

```diff
@@ -96,15 +96,15 @@ interface IDraftableObjectReader<T : Draftable<T>> {
         fetchVersions(publishType, includeDeleted).map(::fetch)
 }
 
-interface IDraftableObjectDao<T : Draftable<T>> : IDraftableObjectReader<T>, IDraftableObjectWriter<T>
+interface ILayoutConceptDao<T : LayoutConcept<T>> : ILayoutConceptReader<T>, ILayoutConceptWriter<T>
```

Comment:
En ole ihan varma tästä `LayoutConcept`-termistä. Mielestäni se aiemmin puhuttu `DraftAware` tms. olisi kuvaavampi, sillä ainakaan mulle `Concept` ei kerro oikein mistä tässä olisi kyse. Ensimmäisenä lukiessa tuli mieleen concept as in concept art, jossa concept-termiä käytetään kuvaamaan enemmän pohjatyönä ja alustavana inspiraation lähteenä muiden tarkemmalle työstölle. Oletan taas että tässä oikeasti pyritään kommunikoimaan että concept = käsite.

Reply:
Se LayoutContextAware on kyllä olemassa ja rajapintana. Tämä on lisätermi "kaikille layoutin käsitteille" ja se toteuttaa sekä tuon LayoutContextAwaren että vähän muutakin sellaista joka ei liity kontekstiin. En itsekään oikein pidä tuosta termistä mutta en ole ainakan toistaiseksi keksinyt parempaakaan :thinking: